### DISCUSSION TITLE
参考文献英文作者名，导师要求仅首字母大写

### DISCUSSION THREAD

**Discussion by Kelipapa** [obster-y/XJTU-thesis#77] [Ideas]
英文作者名默认输出全大写，因此修改了XJTU-thesis.cls
将
\RequirePackage[hyperref=true,backend=biber,defernumbers=true,style=gb7714-015,gbalign=left,gbpub=false]{biblatex}
改为
\RequirePackage[hyperref=true,backend=biber,defernumbers=true,style=gb7714-015,gbalign=left,gbpub=false,**gbnamefmt=lowercase**]{biblatex}

供参考

**Comment by obster-y** [comment]
如果你去搜已经closed的issue，里面已经提到过这一点，同时也提到了要求和国标的自相矛盾

**Reply by Kelipapa** [reply]
> 如果你去搜已经closed的issue，里面已经提到过这一点，同时也提到了要求和国标的自相矛盾

不好意思，不常用GitHub，我以为Issue里面只有四个条目，原来Closed这么多，汗-_-||

**Comment by ghost** [comment]
still, useful share of your snippets.

**Comment by ghost** [comment]
> 英文作者名默认输出全大写，因此修改了XJTU-thesis.cls 将 \RequirePackage[hyperref=true,backend=biber,defernumbers=true,style=gb7714-015,gbalign=left,gbpub=false]{biblatex} 改为 \RequirePackage[hyperref=true,backend=biber,defernumbers=true,style=gb7714-015,gbalign=left,gbpub=false,**gbnamefmt=lowercase**]{biblatex}
> 
> 供参考

a typo `015` -> `2015`
